src/components/Settings/LabelList.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
```

Comment:
Rewrite to SASS, do not use Styled-Components as it doesn't match the stack

Reply:
We changed it from styled-components to SASS